editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2974,6 +2977,42 @@ void CanvasItemEditor::_draw_ruler_tool() {
 		text_pos.x = CLAMP(text_pos.x, text_width / 2, viewport->get_rect().size.x - text_width * 1.5);
 		text_pos.y = CLAMP(text_pos.y, text_height * 1.5, viewport->get_rect().size.y - text_height * 1.5);
 
+		// Draw lines
```

Comment:
```suggestion
		// Draw lines.
```
Same in other comments.
(I know it was like this originally, but you could as well fix it if they are being moved)